aries/agents/rust/aries-vcx-agent/src/helper.rs

```diff
@@ -3,7 +3,7 @@ use aries_vcx::{
     messages::AriesMessage,
     utils::{encryption_envelope::EncryptionEnvelope, from_did_doc_sov_to_legacy},
 };
-use aries_vcx_core::wallet::base_wallet::BaseWallet;
+use aries_vcx_core::wallet2::BaseWallet2;
```

Comment:
Can the original `BaseWallet` trait be removed? If so, why not remove it and rename `BaseWallet2` to `BaseWallet`, and if not, why not?

Reply:
Renamed.